packages/stacked_cli/lib/src/commands/create/create_app_command.dart

```diff
@@ -34,6 +34,18 @@ class CreateAppCommand extends Command {
       help: kCommandHelpV1,
     );
 
+    argParser.addOption(
+      "org",
```

Comment:
Please remove these string literals and follow the pattern of the other options.